app/models/invite.rb

```diff
@@ -0,0 +1,4 @@
+class Invite < ApplicationRecord
+  belongs_to :inviter, class_name: "User", foreign_key: "inviter_id"
+  belongs_to :course, class_name: "Course", foreign_key: "course_id"
```

Comment:
`belongs_to :course` doesn't need the arguments `class_name` and `foreign_key` because it doesn't do anything unconventional.